src/servicex_did_finder_lib/communication.py

```diff
@@ -151,6 +155,9 @@ def add_did_finder_cnd_arguments(parser: argparse.ArgumentParser):
                                           did finder/servicex communication will be added.
     '''
     parser.add_argument('--rabbit-uri', dest="rabbit_uri", action='store', required=True)
+    parser.add_argument('--prefix', dest="prefix", action='store', required=False,
+                        default="",
```

Comment:
This means it comes in as a empty string, but below you have a check on `None` (line 66 of `servicex_adaptor.py`). In the end it doesn't matter, of course, since it is a straight pre-pend, and pre-pending an empty string is... the same as not.